pkg/expr/mathexp/types_test.go

```diff
@@ -89,22 +89,18 @@ func TestSeriesFromFrame(t *testing.T) {
 					Name: "test",
 					Fields: []*data.Field{
 						data.NewField("time", nil, []time.Time{}),
-						data.NewField("value", nil, []float64{}),
+						data.NewField("value", nil, []*float64{}),
 					},
 				},
-				TimeIdx:         0,
-				TimeIsNullable:  false,
-				ValueIdx:        1,
-				ValueIsNullable: false,
 			},
 		},
 		{
 			name: "[]*float, []*time frame should convert",
```

Comment:
Should this now be the following instead?
```suggestion
			name: "[]*float, []time frame should convert",
```

Reply:
Nice catch, thank you!